src/components/Book.js

```diff
@@ -18,6 +36,21 @@ const Book = (props) => {
             <button className="btn" type="button">Edit</button>
           </div>
         </div>
+        <div className="middle-container">
```

Comment:
- [ ] I kindly suggest that your follow the design given on Zeplin exactly because the design uses a blue color for the title, text, and button color background, e.tc. while you used a green color.

## DESIGN:
![bookstore](https://user-images.githubusercontent.com/52934128/185746815-56ca630a-535c-4e3d-ae4a-93a97a034a0c.png)
`

## Your Design
<img width="960" alt="image" src="https://user-images.githubusercontent.com/52934128/185746875-489afee8-37fd-449e-9adf-815d73a74634.png">